query/stdlib/influxdata/influxdb/dependencies.go

```diff
@@ -26,6 +26,9 @@ func (d StorageDependencies) Inject(ctx context.Context) context.Context {
 }
 
 func GetStorageDependencies(ctx context.Context) StorageDependencies {
+	if ctx.Value(dependenciesKey) == nil {
+		return StorageDependencies{}
+	}
```

Comment:
It is possible that I should check the actual `StorageDependencies` struct within the context, but I think that the result will be the same here.